tests/installation/module_registration/register_modules.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Register Application module
+#          in "Extension and Module Selection" dialog
+# Medium:  Online (you see the "Hide Development versions" checkbox)
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    $testapi::distri->get_module_registration()->register_modules([qw(workstation basesystem development legacy containers python3 desktop)]);
+    save_screenshot;
+    my $timeout = 2400 * get_var('TIMEOUT_SCALE', 1);
+    $testapi::distri->get_module_regcode()->wait_regcode_page({
+            timeout => $timeout,
+            interval => 2,
+            message => 'Page to insert module registration code did not appear'});
+    # some modules (e.g. workstation) require registration, provide code
+    my $regcode = get_var('SCC_REGCODE_WE');
+    $testapi::distri->get_module_regcode()->license_workstation_ext_module($regcode);
+    save_screenshot;
+    # confirm to trust the untrusted GPG key
+    $testapi::distri->get_module_regcode()->handle_untrusted_gpg_key();
```

Comment:
s/handle_untrusted_gpg_key/trust_gnupg_key/
as 'handle' is not clear if we handling trusting or not trusting, there are this sort of cases in multiple popups in the installer, we need to reduce the ambiguity without reading the code in the next layer.